src/deploy/functions/release/planner.ts

```diff
@@ -316,9 +316,8 @@ export function checkForIllegalUpdate(want: backend.Endpoint, have: backend.Endp
 export function checkForV2Upgrade(want: backend.Endpoint, have: backend.Endpoint): void {
   if (want.platform === "gcfv2" && have.platform === "gcfv1") {
     throw new FirebaseError(
-      `[${getFunctionLabel(
-        have,
-      )}] Upgrading from GCFv1 to GCFv2 is not yet supported. Please delete your old function or wait for this feature to be ready.`,
+      `[${getFunctionLabel(have)}] Upgrading from 1st Gen to 2nd Gen is not yet supported. ` +
```

Comment:
When do we actually display this?  On deployment of a scheduled function I guess?

Reply:
On `firebase deploy` when we detect that there exists 1st Gen function whose name is same as a 2nd Gen function you have in your source.